magnetic/Generic/defs.h

```diff
@@ -21,7 +21,7 @@
 *
 *     You should have received a copy of the GNU General Public License
 *     along with this program; if not, write to the Free Software
-*     Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111, USA.
+*     Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
```

Comment:
I think it's probably best to leave the original addresses in the copyright headers (this applies to various other files here as well).

These were a well-intentioned change done years ago, but truthfully it's just churn for no gain.  I haven't changed it in Gargoyle because it doesn't matter, but we have a chance to get this repo right, and start it out clean.

On that note, I'd like to request that any changes you do make get squashed into this single commit, so we keep the initial patches clean.  It's fine to force push to this, even though it's already in a pull request.

Reply:
Original addresses: Understood.

One question about squashing commits: do you want me to squash all commits that goes into master? The idea is to have one whole commit in master then?

Reply:
Yes, I think we should get the first patch in as a single unit, so when the history is looked at, it's not a bunch of little changes here and there, but one cohesive unit.  Like, "this is what has to be done to get this to build properly with Glk".

Reply:
Understood. Thank you